### ISSUE TITLE
`Attribute Error` in `plot_*` modules

### ISSUE THREAD

**Issue by RonM700** [AUTODIAL/AutoEIS#64]
AttributeError: module 'autoeis' has no attribute 'metrics'

**Comment by RonM700** [comment]
it can be solved, if you replace metrics by utils
in all respective lines, so please update you Repo. Thanks

**Comment by ma-sadeghi** [comment]
Thanks for reporting this. We'll look into it.

**Comment by RonM700** [comment]
Hi Amin,

I faced problems in PARALLEL processor mode (Apple M1) with the latest Version 0.0.21, as the code just skips the batch function when creating ECMs.

Best,
Ronny

**Comment by ma-sadeghi** [comment]
Hi @RonM700, please open up a new issue with a minimal working example that reproduces the bug so we can follow up properly
